hummingbot/client/ui/completer.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, hummingbot_application):
         self._trading_timeframe_completer = WordCompleter(["infinite", "from_date_to_date", "daily_between_times"], ignore_case=True)
         self._derivative_completer = WordCompleter(AllConnectorSettings.get_derivative_names(), ignore_case=True)
         self._derivative_exchange_completer = WordCompleter(AllConnectorSettings.get_derivative_names().difference(AllConnectorSettings.get_derivative_dex_names()), ignore_case=True)
+        self._amm_arb_connector_completer = WordCompleter(set(AllConnectorSettings.get_connector_settings().keys()).difference(AllConnectorSettings.get_derivative_names()), ignore_case=True)
```

Comment:
Doing this is going to cause conflict with derivative connector.
The only line that need to be changed in this file is line 39:
```
self._spot_completer = WordCompleter(sorted(AllConnectorSettings.get_exchange_names().union(AllConnectorSettings.get_gateway_evm_amm_connector_names())), ignore_case=True)
```

Reply:
I dont think it would be right to include the amm exchanges as a spot connector.